src/agenteval/cli.py

```diff
@@ -120,146 +126,95 @@ def cli():
     help="Score a directory of evaluation logs.",
 )
 @click.argument("log_dir", type=click.Path(exists=True, file_okay=False))
-@click.option(
-    "--config-path",
-    "config_path",
-    type=str,
-    help=f"Path to a yml config file. Ignored if {EVAL_FILENAME} exists.",
-    default=None,
-)
-@click.option(
-    "--split",
-    type=str,
-    help=f"Config data split. Ignored if {EVAL_FILENAME} exists.",
-    default=None,
-)
 def score_command(
     log_dir: str,
-    config_path: str | None,
-    split: str | None,
 ):
-    # Load or create EvalResult and process logs (inlined from processor)
-    json_path = Path(log_dir) / EVAL_FILENAME
-    if json_path.exists():
-        try:
-            raw = json_path.read_text(encoding="utf-8")
-            eval_result = EvalResult.model_validate_json(raw)
-        except Exception as e:
-            raise click.ClickException(
-                f"Failed to load existing '{EVAL_FILENAME}' at {json_path}: {e}"
-            )
-        if config_path:
-            try:
-                cli_cfg = load_suite_config(config_path)
-                if cli_cfg.version != eval_result.suite_config.version:
-                    click.echo(
-                        f"Warning: CLI config version '{cli_cfg.version}' "
-                        f"does not match JSON config version '{eval_result.suite_config.version}'."
-                    )
-            except Exception as e:
-                click.echo(
-                    f"Warning: could not load CLI config '{config_path}' for comparison: {e}"
-                )
-        if split and split != eval_result.split:
-            raise click.ClickException(
-                f"Split mismatch: JSON split '{eval_result.split}' != CLI split '{split}'"
-            )
-    else:
-        if not config_path or not split:
-            raise click.ClickException(
-                "--config-path and --split must be provided when no existing result JSON"
-            )
-        suite_cfg = load_suite_config(config_path)
-        eval_result = EvalResult(suite_config=suite_cfg, split=split)
+    hf_url_match = re.match(HF_URL_PATTERN, log_dir)
+    if hf_url_match is not None:
+        # Download the logs from HF URL
```

Comment:
If `score` is run with a `hf://` URL, download the logs locally and upload score files back to the same repo, under a different path